data_science.md

```diff
@@ -4,6 +4,16 @@ This file contains resources to learn about Data Science and its sub fields.
 
 * Classical Machine Learning
 * Computer Vision
-* Deep Learning
+* [Deep Learning](#deep-learning)
 * Natural Language Processing 
 * Reinforcement Learning
+
+## Deep Learning
+
+### Neural Networks by 3Blue1Brown
+
+A playlist containing 4(four) videos about neural networks and their working. It gives a mathematical intution about neural networks and concepts like Gradient Desent and Backpropogation. 
```

Comment:
'Descent' is spelled wrong. Please double-check for spelling errors.

Reply:
@aviaryan, thanks for the instruction, I have made the requested change.